Revit_Core_Engine/Modify/SetProperties.cs

```diff
@@ -37,7 +37,7 @@ public static partial class Modify
         /****              Public methods               ****/
         /***************************************************/
 
-        public static void SetProperties(this IObject iObject, Element element, ParameterSettings settings = null)
+        public static void SetProperties(this IObject iObject, Element element, MappingSettings settings = null)
```

Comment:
And this one, though this is explicitly doing `settings == null` checks so this method is unlikely to be a culprit.